example/stretch_grasping/README.md

```diff
@@ -0,0 +1,45 @@
+# Stand-alone grasping setup
+
+## Environment setup on server
+```
+# General installation
+conda create -n home_robot python=3.8 && conda activate home_robot
+conda install -c conda-forge mamba
```

Comment:
Mamba only officially supports installing into the base environment. See docs for details: https://mamba.readthedocs.io/en/latest/installation.html

Using `mamba create` also speeds up the env creation process anyways.